api/actions/createWorkItem.js

```diff
@@ -67,28 +79,39 @@ module.exports = {
                                 interval: 60000
                             });
                             if (syncedItem) {
-                                return await Github.createComment({
-                                    req,
-                                    body: `This issue has been linked to a new work item: ${getWorkItemUrl(
-                                        syncedItem,
-                                        hideWorkItemUrl
-                                    )}`
-                                });
+                                return await updateIssue(req, `This issue has been linked to a new work item: ${getWorkItemUrl(syncedItem, hideWorkItemUrl)}`, suppressGithubComments);
                             }
-                            return await Github.createComment({
-                                req,
-                                body:
-                                    'Sorry we could wait until Heroku connect make the syncronization.'
-                            });
+                            return await updateIssue(req, 'Sorry we could wait until Heroku connect make the synchronization.', suppressGithubComments);
                         }
                     }
                 );
             }
-            return await Github.createComment({
-                req,
-                body: getBuildErrorMessage(config, milestone)
-            });
+            return await updateIssue(req, getBuildErrorMessage(config, milestone, suppressGithubComments));
         }
         return null;
     }
 };
+function formatToGus(url, body) {
+    var formattedDescription;
+    remark().use(markdown).use(strip).process(body, (err, file) => {
+        if (err) {
+            throw err;
+        }
+        formattedDescription = 'Github issue link: '.concat(url, '\n', String(file));
+    });
+    return formattedDescription;
+}
+
+async function updateIssue(req, body, suppressGithubComments) {
+    if (suppressGithubComments) {
+        return await Github.updateDescription({
```

Comment:
if suppressGithubComment we should write the comments in the description of the issue, it will be hard to keep in sync/updated the description moving forward. I will say `supressGithubComments` suppress any communication back to Github

Reply:
Let me bring this up with my PM who initially suggested this. FYI @ntotten @smaddox-sf